sdk/core/azure-core/src/test/java/com/azure/core/util/CoreUtilsTests.java

```diff
@@ -40,16 +40,17 @@ public void findFirstOfTypeWithNoneOfType() {
 
     @Test
     public void testProperties() {
-        assertNotNull(CoreUtils.getProperties("azure-core.properties").get("version"));
-        assertNotNull(CoreUtils.getProperties("azure-core.properties").get("name"));
-        assertTrue(CoreUtils.getProperties("azure-core.properties").get("version")
+        assertNotNull(CoreUtils.getUserAgentPropertiesFromProperties("azure-core.properties").getVersion());
+        assertNotNull(CoreUtils.getUserAgentPropertiesFromProperties("azure-core.properties").getName());
+        assertTrue(CoreUtils.getUserAgentPropertiesFromProperties("azure-core.properties").getVersion()
```

Comment:
Should this just make a single call to CoreUtils?

Reply:
Done. Will push in with another commit